user-contrib/Ltac2/tac2ffi.ml

```diff
@@ -318,6 +320,17 @@ let open_ = {
   r_id = false;
 }
 
+let of_uint63 n = ValUint63 n
+let to_uint63 = function
+| ValUint63 n -> n
+| _ -> assert false
+
+let uint63 = {
+  r_of = of_uint63;
+  r_to = to_uint63;
+  r_id = false;
```

Comment:
What's r_id for?

Reply:
Future-proof code, in case we switch to a physical identity representation for values (i.e. `Obj.t`). That way you know you don't have to recursively convert the subcomponents if they all have the `r_id` flag set.